package.json

```diff
@@ -63,7 +63,7 @@
     "unassertify": "^2.0.3"
   },
   "peerDependencies": {
-    "mapbox-gl": ">=0.27.0 <=0.31.0"
+    "mapbox-gl": ">=0.27.0 <=0.32.1"
```

Comment:
I wouldn't recommend that. If you look at the release notes you will see occasional minor breaking changes. Officially semver only suggests it is safe to auto-update within patch releases before v1.